javaee.wildfly/nbproject/project.properties

```diff
@@ -73,7 +73,7 @@ ${j2ee.common.dir}/modules/org-netbeans-modules-j2ee-common.jar:\
 ${project.libraries.dir}/modules/org-netbeans-modules-project-libraries.jar:\
 ${projectapi.dir}/modules/org-netbeans-modules-projectuiapi.jar:\
 ${javaee.wildfly.dir}/modules/org-netbeans-modules-javaee-wildfly.jar:\
-${javaee.wildfly.dir}/modules/ext/jsr88javax.jar:\
+#${javaee.wildfly.dir}/modules/ext/deployment-api-1.2-rev-1.jar:\
```

Comment:
Why was this commented out?

Reply:
My mistake I was checking whether module really depends on the jar or not. I will correct it